tools/sqf_validator.py

```diff
@@ -83,8 +85,8 @@ def popClosing():
                         if (c == '"' or c == "'"):
                             isInString = True
                             inStringType = c
-                        elif (c == '#'):
-                            checkForSemiColumn = False
+                        elif (c == '#' and onlyWhitespace):
+                            checkForSemiColon = False
```

Comment:
```suggestion
                            ignoreTillEndOfLine = True
```
Was just ignored since a774fd1835b36dde0d28d5b367bd25de774ecef9.